client/src/main/java/org/apache/rocketmq/client/Validators.java

```diff
@@ -60,6 +60,7 @@ public static void checkMessage(Message msg, DefaultMQProducer defaultMQProducer
         // topic
         Validators.checkTopic(msg.getTopic());
         Validators.isNotAllowedSendTopic(msg.getTopic());
+        Validators.isSystemTopic(msg.getTopic());
```

Comment:
This may affect the sending of trace messages. 
https://github.com/apache/rocketmq/blob/7281b5d9a0db15e5ec1ab9e3444091e7022f65fb/client/src/main/java/org/apache/rocketmq/client/trace/AsyncTraceDispatcher.java#L386

Reply:
You are right! The topic RMQ_SYS_TRACE_TOPIC should be excluded.